qa-dcf.planx-pla.net/manifest.json

```diff
@@ -7,22 +7,22 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2020.11",
+    "ambassador": "quay.io/datawire/ambassador:1.4.2",
+    "arborist": "quay.io/cdis/arborist:2021.11",
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
-    "datareplicate": "quay.io/cdis/dcf-dataservice:2020.11",
-    "fence": "quay.io/cdis/fence:2020.11",
-    "indexd": "quay.io/cdis/indexd:2020.11",
-    "peregrine": "quay.io/cdis/peregrine:2020.11",
-    "pidgin": "quay.io/cdis/pidgin:2020.11",
-    "revproxy": "quay.io/cdis/nginx:2020.11",
-    "sheepdog": "quay.io/cdis/sheepdog:2020.11",
-    "portal": "quay.io/cdis/data-portal:2020.11",
+    "datareplicate": "quay.io/cdis/dcf-dataservice:2021.11",
+    "fence": "quay.io/cdis/fence:2021.11",
+    "indexd": "quay.io/cdis/indexd:2021.11",
+    "peregrine": "quay.io/cdis/peregrine:2021.11",
+    "pidgin": "quay.io/cdis/pidgin:2021.11",
+    "revproxy": "quay.io/cdis/nginx:2021.11",
+    "sheepdog": "quay.io/cdis/sheepdog:2021.11",
+    "portal": "quay.io/cdis/data-portal:2021.11",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "sower": "quay.io/cdis/sower:2020.11",
-    "metadata": "quay.io/cdis/metadata-service:2020.11",
-    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.11",
-    "dashboard": "quay.io/cdis/gen3-statics:2020.11",
-    "status-api": "quay.io/cdis/status-dashboard:2020.11"
+    "sower": "quay.io/cdis/sower:2021.11",
+    "metadata": "quay.io/cdis/metadata-service:2021.11",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2021.11",
+    "guppy": "quay.io/cdis/status-dashboard:2021.11"
```

Comment:
this doesnt look right to me! @BinamB

Reply:
yeah this is definitely wrong, bad copy paste or something. let me fix